src/utils/user_actions.cljs

```diff
@@ -0,0 +1,18 @@
+(ns utils.user-actions
```

Comment:
Thank you @flexsurfer for your review.
Usually, we use events namespace to define re-frame events and this namespace will hold only `highly reusable dispatch functions`. The name might mislead. wdyt?

Reply:
events-helper? :)

Reply:
Thank you! I now understand why we want to avoid using user-actions. It is also too specific and suggests that the namespace will only contain dispatches related to user actions.

Updated docstring:
```
Provides highly reusable dispatch functions to ensure consistency and avoid duplication in managing various events,
including navigation and other actions that affect application behavior.
```

For the namespace name, any of the following options work for me:

- events-helper/navigate-back
- event-dispatch/navigate-back
- event-dispatcher/navigate-back

Reply:
Using `events-helper`, and will push changes. Please feel free to let me know if you have any more suggestions.

Reply:
done